src/segmantic/seg/monai_unet.py

```diff
@@ -548,3 +669,182 @@ def print_table(header, vals, indent="\t"):
+    label_files = sorted(Path(labels_dir).glob("*.nii.gz"))
+
+    image_idx = np.arange(len(image_files))
+
+    cv_split = Path(Path(image_dir).parent) / 'CV_Split'
+    cv_split_train = cv_split / 'train'
+    cv_split_predict = cv_split / 'predict'
+    cv_split_train_img = cv_split_train / 'images'
+    cv_split_train_lbl = cv_split_train / 'labels'
+    cv_split_predict_img = cv_split_predict / 'images'
+    cv_split_predict_lbl = cv_split_predict / 'labels'
+
+    all_paths = [cv_split, cv_split_train, cv_split_predict,
+                 cv_split_train_img, cv_split_train_lbl, cv_split_predict_img, cv_split_predict_lbl]
+
+    for path in all_paths:
+        if not path.exists():
+            path.mkdir()
+
+    for config_file in Path(config_files_dir).iterdir():
+        output_dir_scenario = output_dir.joinpath(str(config_file.name))
+        if not output_dir_scenario.exists():
+            output_dir_scenario.mkdir()
+
+        for train_idx, test_idx in kf.split(image_idx, image_idx):
+
+            current_output = output_dir_scenario.joinpath(str(test_idx))
+            print(current_output)
+            if current_output.exists():
+                continue
+            else:
+                current_output.mkdir()
+
+            if config_file.exists():
+                data = json.loads(config_file.read_text())
+                data["image_dir"] = str(cv_split_train_img)
+                data["labels_dir"] = str(cv_split_train_lbl)
+                data["output_dir"] = str(current_output)
+                current_layers = data["layers"]
+                current_strides = data["strides"]
+            with open(config_file, 'w') as f:
+                json.dump(data, f, indent=4)
+
+            # Delete all files from the temporary file dirs
+            for folder in all_paths:
+                if folder.match('images') or folder.match('labels'):
+                    for file in folder.iterdir():
+                        file.unlink()
+
+            # set new splits
+            image_train = [image_files[index] for index in train_idx]
+            image_test = [image_files[index] for index in test_idx]
+            label_train = [label_files[index] for index in train_idx]
+            label_test = [label_files[index] for index in test_idx]
+
+            for file in image_train:
+                shutil.copyfile(file, cv_split_train_img.joinpath(file.name))
+            for file in image_test:
+                shutil.copyfile(file, cv_split_predict_img.joinpath(file.name))
+            for file in label_train:
+                shutil.copyfile(file, cv_split_train_lbl.joinpath(file.name))
+            for file in label_test:
+                shutil.copyfile(file, cv_split_predict_lbl.joinpath(file.name))
+            print('start training')
+
+            result = sp.run([sys.executable,
+                             "F:/Melanie/ETH/11.Semester/MasterThesis/Python/src/segmantic/scripts/run_monai_unet.py",
```

Comment:
you can obtain the current file path like so `Path(__file__).resolve()` without using hard-coded paths.